indico/htdocs/js/indico/modules/abstracts.js

```diff
@@ -39,6 +39,7 @@
     };
 
     global.setupAbstractPage = function setupAbstractPage() {
+        $('.dropdown-abstract-pdf').parent().dropdown();
```

Comment:
I haven't thought about that. I've just checked again and the dropdown stops working when e.g somebody leaves a comment. Do you know how can I solve it?